css/hoot/components/managePanel/_taskingManager.scss

```diff
@@ -0,0 +1,167 @@
+@import '../../modules/variables';
+
+#taskingManagerPanel {
+    .panel-wrapper {
+        overflow: hidden;
+    }
+}
+
+.taskingManager-table {
+    display: inline-block;
+
+    width: 100%;
+    height: calc(100% - 100px);
+    box-shadow: 0 3px 10px $dark-opacity10;
+    overflow: auto;
+
+    &::-webkit-scrollbar {
+        width: 13px;
+        border-left: 1px solid #DDD;
+    }
+
+    .task-title {
+        display: inline-block;
+    }
+}
+
+.taskingManager-projects,
+.taskingManager-tasks{
+    position: relative;
+    width: 50%;
+    height: 100%;
+    padding: 30px;
+    display: inline-block;
+
+}
+
+.taskingManager-item {
+    padding: 5px 10px 5px 15px;
+    border-bottom: 1px solid $dark-opacity20;
+
+    &:first-child {
+        border-top: none;
+    }
+
+    .taskingManager-header {
+        margin-bottom: 5px;
+    }
+
+    .taskingManager-body {
+
+        label {
+            font-weight: 500;
+        }
+    }
+
+    .taskingManager-description {
+        margin-right: 10px;
+
+        label {
+            display: inline;
+        }
+    }
+
+    .taskingManager-details {
+        label {
+            display: inline;
+        }
+    }
+}
+
+
+.taskingManager-tasks {
+    position: relative;
+    height: 100%;
+    padding: 30px;
+    display: inline-block;
+
+    .taskHeader {
+        .taskHeader-title {
+            display: inline-block;
+            font-size: 16px;
+            line-height: 1.25;
+            font-weight: bold;
+            margin-bottom: 10px;
+        }
+
+        .runAllBtn {
+            float: right;
+            margin-right: 25px;
+            margin-bottom: 5px;
+            height: 35px;
+            width: 85px;
+        }
+
+        .changeset-control {
+            float: right;
+            margin-right: 15px;
+
+            input {
+                height: 35px;
+            }
+        }
+    }
+
+    .taskingManager-item {
+        position: relative;
+        padding: 5px 10px 5px 15px;
+        height: 50px;
+        font-size: 15px;
+
+        &[status="Done"] {
+            background-color: #ffdca3;
+        }
+
+        &[status="Validated"] {
+            background-color: #a3cfa3;
+        }
+
+        &[status="Invalidated"] {
+            background-color: #ffa3a3;
+        }
+
+        &[status="Locked"],
+        &[status="Running"] {
+            border: 4px solid #ffdca3;
+
+            .taskingManager-action-buttons {
+                margin-right: 10px;
+            }
+        }
+
+        &[status=""] {
+            background-color: #cdcdcd;
```

Comment:
maybe a bit lighter gray?  did we even consider white?

Reply:
I like the lighter gray more because in tasking manager the fill is a light grey when the task is not locked and hasn't been worked on